autoload/neomake/makers/ft/go.vim

```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#go#EnabledMakers()
-    return ['go', 'golint', 'govet']
+    return ['go', 'golint', 'govet', 'gometalinter']
```

Comment:
In case `gometalinter` is supposed to replace the others, you should only return it here, in case it is available:

```vim
return executable('gometalinter') ? ['gometalinter'] : ['go', 'golint', 'govet']
```

But I can see that `go` might just catch syntax errors etc.